console/my-account/reference-content/products-availability.mdx

```diff
@@ -36,7 +36,7 @@ Scaleway products are available in multiple regions and locations worldwide. Thi
 | **Managed Databases** | PostgreSQL - MySQL                    | Across Paris region    | Across Amsterdam region | Across Warsaw region   |
 |                       | Redis™                                | PAR1, PAR2             | AMS1, AMS2              | WAW1                   |
 | **Network**           | Private Networks                      | Across Paris region    | Across Amsterdam region | Across Warsaw region   |
-|                       | Public Gateway                        | PAR1, PAR2             | AMS1, AMS2              | WAW1, WAW2             |
+|                       | Public Gateway                        | PAR1, PAR2             | AMS1, AMS2, AMS2        | WAW1, WAW2, WAW3       |
```

Comment:
Shouldn't it me AMS3 ? Or is it intended?

Reply:
Ohhh glad you spotted that, thank you